contracts/AGENTS.md

```
@@ -1,19 +1,169 @@
+sozo migrate --profile sepolia    # Deploy to Sepolia
+sozo migrate --profile mainnet    # Deploy to Mainnet
+sozo migrate --profile slot       # Deploy to Slot testnet
+```
 
-## Commit & Pull Request Guidelines
-Match the existing history: concise, present-tense subjects such as `fix client trait calculation`; add context lines if rationale is non-obvious, and append PR numbers in parentheses when applicable. For pull requests, describe the gameplay or contract impact, note required migrations or manifest updates, list manual test commands (`scarb test`, deploy scripts), and link the tracked issue. Screenshots or log excerpts are welcome when behavior changes.
+## Project Structure
 
-## Deployment & Configuration Tips
-Keep profile-specific settings in the corresponding `dojo_*.toml` files and update manifests when new systems or models are introduced. Regenerate SVG or other assets via `scripts/generate_svg.sh` whenever constants affecting visuals change. Verify world permissions in tests using the helpers in `utils/setup_denshokan.cairo` before promoting changes to shared environments.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)